### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**

### CODE REVIEW

**Review by rtkachuk-moon — APPROVED** [review]
👍

File: src/convertToObject.js [hunk 1]
```diff
@@ -12,7 +12,24 @@
...
+
+    cssProperties.forEach(property => {
+      const part = property.split(':');
+      const key = part[0].trim();
+      const value = part[1] ? part[1].trim() : '';
+
+      if (key !== '') {
+        propertiesObject[key] = value;
+      }
+    });
```
Comment (DanilWeda) [inline src/convertToObject.js:29]: Try to use a method that returns an object!

File: src/convertToObject.js [hunk 2]
```diff
@@ -12,7 +12,24 @@
...
+  const cssValues = sourceString.split('\n');
+  const propertiesObject = {};
+
+  cssValues.forEach(values => {
+    const cssProperties = values.split(';');
+
+    cssProperties.forEach(property => {
+      const part = property.split(':');
+      const key = part[0].trim();
+      const value = part[1] ? part[1].trim() : '';
```
Comment (DanilWeda) [inline src/convertToObject.js:24]: as alternative 
```suggestion
      const [key, value] = part;
```

**Review by DanilWeda — CHANGES_REQUESTED** [review]
Good job! 
Check the test suite and some comments for improvement! 
Thanks!

**Review by olya-shyian — APPROVED** [review]
Wow, impressive!🔥

**Pull request closed without merging**